bundler/lib/bundler/man/bundle-config.1.ronn

```diff
@@ -289,6 +223,71 @@ learn more about their operation in [bundle install(1)](bundle-install.1.html).
 * `without` (`BUNDLE_WITHOUT`):
    A space-separated or `:`-separated list of groups whose gems bundler should not install.
 
+## REMEMBERING OPTIONS
+
+Flags passed to `bundle install` or the Bundler runtime, such as `--path foo` or
+`--without production`, are remembered between commands and saved to your local
+application's configuration (normally, `./.bundle/config`).
+
+However, this will be changed in bundler 3, so it's better not to rely on this
+behavior. If these options must be remembered, it's better to set them using
+`bundle config` (e.g., `bundle config set --local path foo`).
+   
+The options that can be configured are:
+
+* `--bin`:
```

Comment:
Can we keep using the setting name without `--` here? I think the docs are referring to the options that can be configured, not to the remembered flags that can be passed to the CLI.